aea/configurations/base.py

```diff
@@ -573,7 +571,6 @@ class ConnectionConfig(ComponentConfiguration):
     FIELDS_ALLOWED_TO_UPDATE: FrozenSet[str] = frozenset(
         ["config", "cert_requests", "is_abstract", "build_directory"]
     )
-    FIELDS_WITH_NESTED_FIELDS: FrozenSet[str] = frozenset(["config"])
```

Comment:
Both config and cert_requests have nesting, so not sure why we ever had it this way.